### PR TITLE
Features/keyboard

### LINKED ISSUE #5: Keyboard
Adding a keyboard option will require some thinking outside the box. iOS is really nice with the setOverrideUserInterfaceStyle method as it provides a fallback - 0 - that returns control to iOS. The keyboard does not have this, or so my limited research seems to show. This may require rethinking Dawn's standard implementation in the case of the keyboard feature to allow seamless changes while also allowing control to be given back to the app/system.

### PR DESCRIPTION
**Pull request by tomaszpoliszuk**

### CODE REVIEW

File: Tweak.xm [hunk 1]
```diff
@@ -334,20 +329,16 @@ NSInteger apps;
...
-//    }
-//    else {
-//        %orig;
-//    }
-//}
-//%end
+#pragma mark - Keyboard
+%hook UITextInputTraits
+- (long long)keyboardAppearance {
+    long long origValue = %orig;
```
Comment (EthanRDoesMC) [inline Tweak.xm]: FYI: Unless there was an issue, this is unnecessary; returning %orig will do the trick. The rest of Dawn returns %orig so let’s keep it unified :)

Comment (EthanRDoesMC) [reply Tweak.xm]: Remove this line — don’t need it.

File: Tweak.xm [hunk 2]
```diff
@@ -334,20 +329,16 @@ NSInteger apps;
...
+    if ( keyboard > 0 ) {
+        return keyboard;
+    }
+    return origValue;
```
Comment (EthanRDoesMC) [inline Tweak.xm]: replace with `return %orig;`

**Review by EthanRDoesMC — CHANGES_REQUESTED** [review]
A little bit of streamlining/standardizing for the method return

**Comment by EthanRDoesMC** [comment]
Not used to getting pull requests; can I not add a commit of my own? Else I would've just fixed it myself (trying to prevent merge conflicts)

**Comment by tomaszpoliszuk** [comment]
No problem at all - just sent updated version.

Have a nice day!

**Comment by EthanRDoesMC** [comment]
Looks good! I'll go ahead and merge it, but there is a slight problem: localizing "Keyboard". However, the settings app has a panel named Keyboard, so I'll see if it can't pull from that. Or I'll just copy the name manually by switching device languages, lol.

**Review by EthanRDoesMC — APPROVED** [review]
Add translations before uploading to BigBoss. Otherwise, 10/10.

**Pull request merged**